### PR TITLE
Event addition ir

### PR DESCRIPTION
**Pull request by Raggetti**
Addition of 50 Events and 2 items to trade at shop.

### CODE REVIEW

File: Resources/GameFiles/Events/EVENT_BankSneeze.xml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+				<EventOptionDescription>Ask someone for a tissue</EventOptionDescription>
+				<EventOptionPostDescription>You quickly ask someone for a tissue, they hand you one and you sneeze into it.</EventOptionPostDescription>
+				<EventOptionImmunityModification>0</EventOptionImmunityModification>
+				<EventOptionContagionLevelModifier>10</EventOptionContagionLevelModifier>
+				<EventOptionKarmaModification>0</EventOptionKarmaModification>
+			</EventOption>
+			<EventOption>
+				<EventOptionID>BankSneezeElbow</EventOptionID>
+				<EventOptionDescription>Sneeze into your elbow</EventOptionDescription>
+				<EventOptionPostDescription>[Option2Consequence]</EventOptionPostDescription>
```
Comment (RiggsDan) [inline Resources/GameFiles/Events/EVENT_BankSneeze.xml:20]: Needs to be replaced

File: Resources/GameFiles/Events/EVENT_HomeFriendsForFootball.xml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+		<IsPassiveEvent>false</IsPassiveEvent>
+		<IsSingleOccurence>true</IsSingleOccurence>
+		<EventOptions>
+			<EventOption>
+				<EventOptionID>HomeFriendsForFootballGo</EventOptionID>
+				<EventOptionDescription>Go over and watch the game.</EventOptionDescription>
+				<EventOptionPostDescription>It was nice to see your mates after so long although your team lost :(</EventOptionPostDescription>
+				<EventOptionImmunityModification>0</EventOptionImmunityModification>
+				<EventOptionContagionLevelModifier>20</EventOptionContagionLevelModifier>
+				<EventOptionKarmaModification></EventOptionKarmaModification>
```
Comment (RiggsDan) [inline Resources/GameFiles/Events/EVENT_HomeFriendsForFootball.xml:15]: Missing Karma modification

File: Resources/GameFiles/Events/EVENT_HomeNeighbourFlour.xml [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+					<RequiredItem>
+						<ItemID>FlourID</ItemID>
+						<ItemCount>1</ItemCount>
+					</RequiredItem>
+				</RequiredItems>
+			</EventOption>
+			<EventOption>
+				<EventOptionID>HomeNeighbourFlourNone</EventOptionID>
+				<EventOptionDescription>Tell them you don't have any flour to give and apologise.</EventOptionDescription>
+				<EventOptionPostDescription></EventOptionPostDescription>
```
Comment (RiggsDan) [inline Resources/GameFiles/Events/EVENT_HomeNeighbourFlour.xml:34]: Remove this tag (As optional)

File: Resources/GameFiles/Events/EVENT_HospitalTakeNeighbourHome.xml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+		<IsPassiveEvent>false</IsPassiveEvent>
+		<IsSingleOccurence>true</IsSingleOccurence>
+		<EventOptions>
+			<EventOption>
+				<EventOptionID>HospitalTakeNeighbourHomeTake</EventOptionID>
+				<EventOptionDescription>Take neighbour home</EventOptionDescription>
+				<EventOptionPostDescription>You drive your neighbour home. They thank you with a box of chocolates.</EventOptionPostDescription>
+				<EventOptionImmunityModification>0</EventOptionImmunityModification>
+				<EventOptionContagionLevelModifier>5</EventOptionContagionLevelModifier>
+				<EventOptionKarmaModification></EventOptionKarmaModification>
```
Comment (RiggsDan) [inline Resources/GameFiles/Events/EVENT_HospitalTakeNeighbourHome.xml:15]: Missing value

File: Resources/GameFiles/Events/EVENT_BankSneeze.xml (hunk 1 above)
Comment (Raggetti) [reply Resources/GameFiles/Events/EVENT_BankSneeze.xml]: This has been corrected.

File: Resources/GameFiles/Events/EVENT_HomeFriendsForFootball.xml (hunk 2 above)
Comment (Raggetti) [reply Resources/GameFiles/Events/EVENT_HomeFriendsForFootball.xml]: This has been corrected.

File: Resources/GameFiles/Events/EVENT_HomeNeighbourFlour.xml (hunk 3 above)
Comment (Raggetti) [reply Resources/GameFiles/Events/EVENT_HomeNeighbourFlour.xml]: This has been corrected.

File: Resources/GameFiles/Events/EVENT_HospitalTakeNeighbourHome.xml (hunk 4 above)
Comment (Raggetti) [reply Resources/GameFiles/Events/EVENT_HospitalTakeNeighbourHome.xml]: This has been corrected.

**Pull request merged**